Assets/BossRoom/Scripts/Server/Game/Entity/ServerWaveSpawner.cs

```diff
@@ -63,8 +67,19 @@ public class ServerWaveSpawner : NetworkBehaviour
         [SerializeField]
         float m_ProximityDistance;
         [SerializeField]
-        [Tooltip("The spawner won't create more than this many entities at a time. 0 = don't track spawn count")]
-        int m_MaxActiveSpawns;
+        [Tooltip("When looking for players within proximity distance, should we count players in stealth mode?")]
+        bool m_DetectStealthyPlayers;
+
+        [Header("Spawn Cap (i.e. number of simultaneously spawned entities)")]
+        [SerializeField]
+        [Tooltip("The minimum number of entities this spawner will try to maintain (regardless of player count)")]
+        int m_MinSpawnCap;
```

Comment:
Could we initialize these values to some default ones in code, please?

Reply:
- Assigned defaults (based on the settings in the spawner prefab, but with a few modifications)
- Moved some non-serialized variables down below the massive list of serialized variables (just for clarity)

ok